app/src/lib/i18n/ui-text.js

```diff
@@ -255,6 +257,9 @@ const uiText = {
     aboutAppDescription:
       "La plateforme Fiji Department of Water & Sewerage (DWS) DataPro est conçue pour améliorer la gestion des services d'eau et d'assainissement aux Fidji.",
     appVersionLabel: 'Version de l’application',
+    emptySubmissionMessage1: 'Aucune donnée n’a été collectée avec ce formulaire pour le moment',
```

Comment:
Small thing, but if you are going to give meaning to your variable name, I would do it all the way:

- emptySubmissionMessage1 => emptySubmissionMessageInfo
- emptySubmissionMessage2 => emptySubmissionMessageAction

It contributes making the code even more readidable.

Reply:
Noted, I've fixed those variable names